engine/setup/CMakeLists.txt

```diff
@@ -31,4 +31,4 @@ IF (NOT BEOS)
         ${GTK3_INCLUDE_DIRS}
     )
 	TARGET_LINK_LIBRARIES(vegasettings ${GTK_LIBS})
-ENDIF (NOT BEOS)
+ENDIF ((NOT BEOS) AND (NOT WIN32))
```

Comment:
It would be good if we had an else clause that emitted a message about why we're skipping, something like:
```
ELSE ((NOT BEOS) AND (NOT WIN32))
    MESSAGE("SKIPPING vegasettings: Vega Settings Not support on the Platform at this time")
ENDIF ((NOT BEOS) AND (NOT WIN32))
```

Reply:
Done.